src/etc/inc/gwlb.inc

```diff
@@ -206,6 +300,7 @@ function setup_gateways_monitor() {
 		echo "Setting up gateway monitors...";
 	}
 	$monitor_ips = array();
+	$GLOBALS['inuse_static_routes'][] = array();
```

Comment:
I take it the purpose of this is to avoid removing a static route configured by the user. If so, I don't think it's necessary. When a static route is configured by the user, it stays when dpinger restarts even when "Do not add static route..." is checked on the gateway. Regardless, I see this more of a user configuration issue than something we should worry about in the code.